### PR TITLE
Fixes for nocontrolsamples

### PR DESCRIPTION
**Pull request by kubranarci**
- add cram/crai files to input list (assets/schema_input.json)
- add only a warn about the indices does not exists in converttovcf.json (bin/convertToStdVCF.py)
- change default arg for ENSEMBL VEP (conf/modules.config)
- set TMPDIR to working dir (modules/local/check_if_corrupted.nf)
- edit nextflow version to latest (nextflow.config)

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -140,7 +140,7 @@ process {
         ]
     }
     withName: 'ENSEMBLVEP_VEP' {
-        //ext.args         ='--everything --filter_common --per_gene --total_length --offline'
+        ext.args         ='--everything --filter_common --per_gene --total_length --offline'
```
Comment (NagaComBio) [inline conf/modules.config:143]: Same as in SNV workflow. We have to leave out `--filter_common --per_gene`

**Review by NagaComBio — CHANGES_REQUESTED** [review]
We need a changelog for the patch and an update for the VEP options.

**Comment by kubranarci** [comment]
> We need a changelog for the patch and an update for the VEP options.

I will do that

File: conf/modules.config (hunk 1 above)
Comment (kubranarci) [reply conf/modules.config]: done

**Review by NagaComBio — APPROVED** [review]

**Pull request merged**